content/versions.yml

```diff
@@ -1,3 +1,5 @@
+- title: '16.8'
+  changelog: https://github.com/facebook/react/blob/master/CHANGELOG.md#1680-february-6-2019
 - title: '16.7'
```

Comment:
We'll want to add a `path` and `url` for the 16.7 docs too. I can help with that bit.

Reply:
Feel free to add them :)